Source/Blazorise/Base/BaseComponent.cs

```diff
@@ -690,6 +695,21 @@ public TextTransform TextTransform
         }
     }
 
+    /// <summary>
+    /// Gets or sets the text decoration.
+    /// </summary>
+    [Parameter]
+    public TextDecoration TextDecoration
+    {
+        get => textDecoration;
+        set
+        {
+            textDecoration = value;

```

Comment:
You usually have the approach of checking if the value == existingvalue. Is it applicable here?

I was thinking, does it make sense to have an helper on the base component that does this? Not sure if it makes sense, just throwing the idea out there.
setProperty(field, newvalue){
if (field==newValue)

....
DirtyClasses()
}

Reply:
Done.

Not sure how it would that work unless we use something like reflection. Better to stick with the manual approach.

Reply:
Hmm probably can make the method generic or overloads for the common cases. Don't think reflection is needed.